spec/money_column_spec.rb

```diff
@@ -60,15 +60,21 @@ class MoneyWithReadOnlyCurrency < ActiveRecord::Base
   it 'handles legacy support for saving floats' do
     record.update(price: 3.21, prix: 3.21)
     expect(record.price.value).to eq(3.21)
+    expect(record.price.currency_iso).to eq(currency)
     expect(record.price_usd.value).to eq(3.76)
+    expect(record.price_usd.currency_iso).to eq('USD')
     expect(record.prix.value).to eq(3.21)
+    expect(record.prix.currency_iso).to eq('CAD')
```

Comment:
We might want to remove the `prix` column now that we've made the column name explicit all the time, it's not different anymore.

Reply:
it does have a different currency, i'll have a look 👍